cgroups/fs/devices.go

```diff
@@ -32,6 +32,16 @@ func (s *DevicesGroup) Set(path string, cgroup *configs.Cgroup) error {
 				return err
 			}
 		}
+	} else {
```

Comment:
@LK4D4 Could you give more explanation about why should not use return nil and else?